okhttp/src/jvmMain/kotlin/okhttp3/ConnectionSpec.kt

```diff
@@ -51,6 +51,15 @@ class ConnectionSpec internal constructor(
   private val tlsVersionsAsString: Array<String>?
 ) {
 
+  /**
+   * Returns the cipher suites to use for a connection. Returns null if all of the SSL socket's
+   * enabled cipher suites should be used.
+   */
+  @get:JvmName("cipherSuitesAsString") val cipherSuitesAsStringList: List<String>?
+    get() {
+      return cipherSuitesAsString?.let { Collections.unmodifiableList(it.asList()) }
```

Comment:
I wonder why we need this, since `CipherSuite.forJavaName()` should create them on the fly.

See https://github.com/square/okhttp/blob/afcc2df23aa855220f0d532b404ec393d927520c/okhttp/src/jvmMain/kotlin/okhttp3/ConnectionSpec.kt#L60

and 

https://github.com/square/okhttp/blob/afcc2df23aa855220f0d532b404ec393d927520c/okhttp/src/jvmMain/kotlin/okhttp3/CipherSuite.kt#L426